build/yaml/jobs/windows/postbuild.yaml

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# Job Template: PostBuild steps
+
+jobs:
+- job: PostBuild
+  displayName: PostBuild
+
+  variables:
+    Packaging.EnableSBOMSigning: true
+
+  pool:
+    name: VSEngSS-MicroBuild2019-1ES
+    demands:
+    - msbuild
+    - vstest
+
+  workspace:
+    clean: outputs
+
+  steps:
+  - checkout: none
+
+  - template: ../../steps/azuredevops/downloadbuildartifacts.yaml
+    parameters:
+      ArtifactName: binaries-windows-Release
+
+  - template: ../../steps/microbuild/sbom.yaml
```

Comment:
We need more than just the Windows binaries; we definitely need the Linux binaries.

Reply:
assuming we just need to target packages, then I can pull down packages-all which contains both platform files.